stable/aws-calico/README.md

```diff
@@ -0,0 +1,44 @@
+# Calico on Amazon EKS
+
+This chart installs Calico on Amazon EKS: https://docs.aws.amazon.com/eks/latest/userguide/calico.html
+
+## Prerequisites
+
+- Kubernetes 1.11+ running on AWS
+
+## Installing the Chart
+
+First add the EKS repository to Helm:
+
+```shell
+helm repo add eks https://aws.github.io/eks-charts
+```
+
+To install the chart with the release name `aws-calico` and default configuration:
+
+```shell
+$ helm install --name aws-calico --namespace kube-system eks/aws-calico
+```
+
+To install into an EKS cluster where the CNI is already installed, you can run:
+
+```shell
+helm upgrade --install --recreate-pods --force aws-calico --namespace kube-system eks/aws-calico
+```
+
+If you receive an error similar to `Error: release aws-calico failed: <resource> "aws-node" already exists`, simply rerun the above command.
+
+## Configuration
+
+The following table lists the configurable parameters for this chart and their default values.
+
+| Parameter               | Description                                             | Default            |
+| ------------------------|---------------------------------------------------------|--------------  ----|
+| `calico.typha.image`    | Calico Typha Image                                      | ``                 |
+| `calico.node.image`     | Calico Node Image                                       | ``                 |
```

Comment:
The default values for these configuration options are `quay.io/calico/typha:v3.8.1` and `quay.io/calico/node:v3.8.1`

Reply:
changed